src/ZODB/Connection.py

```diff
@@ -1020,9 +1018,10 @@ def _cache_items(self):
         items = self._cache.lru_items()
         # fine everything. some on the lru list, some not
         everything = self._cache.cache_data
-        # remove those items that are on the lru list
+        # remove those items that are on the lru list (which may not actually
+        # be in the full cache, under the Python implementation)
```

Comment:
Ah. No, it doesn't seem to be anymore. At least, the tests that failed before that change aren't failing if I revert it. If it does fail, it means that the picklecache is corrupted, and we have a CFFI level memory leak (under PyPy).